src/index.html

```diff
@@ -7,5 +7,14 @@
   </head>
   <body>
     <div id="houston"></div>
+
+    <script>
```

Comment:
This script should go in the head.

It also should 

- only show the GA code when APP_ENV !== development https://github.com/ampedandwired/html-webpack-plugin/tree/03758f46ccadcc9e0b90aac9898ffce46a8821f2#writing-your-own-templates
- The GA Property code (UA-83921870-1) should be configurable